support/bin/silver-custom

```diff
@@ -32,4 +32,16 @@ fi
 # Set flags if not overriden in environment
 SVJVM_FLAGS=${SVJVM_FLAGS:-"-Xmx5000M -Xss20M"}
 
-java $SVJVM_FLAGS -jar "$SILVER" "$@" && ant
+# Save the command line arguments to pass to Silver
+SV_ARGS="$@"
+
+# Tell ant if we named the build file something else
+while [ "$#" -gt 0 ]; do
+    if [ "$1" = --build-xml-location ]; then
```

Comment:
should this be a `case` with a `--build-xml-location=*` clause to catch that form too, or will that blow up for other reasons

Reply:
This is parsing a flag that is also passed when invoking the Silver jar, and our `cmdargs` library doesn't currently support that format.